css-animations-2/Overview.bs

```diff
@@ -167,6 +167,7 @@ last) as follows:
     *  element
     *  ::marker
     *  ::before
+    *  other pseudo-elements in alphabetical order by selector
```

Comment:
This seems reasonable to me, but I defer to if @birtles has any thoughts.

Editorial comment: I'm not expert, but I know web-animations uses the verbage: 'sorted in ascending order by the Unicode codepoints', so that may be preferable here (e.g. 'any other pseudo-elements not mentioned specifically in this list, sorted in ascending order by the Unicode codepoints that make up each selector').

Reply:
done